### DISCUSSION TITLE
stream aspect ratio

### DISCUSSION THREAD

**Discussion by arpa-git** [streamlink/streamlink-twitch-gui#777] [Q&A] [answered]
I seem to have a problem with the bottom of my screen; when I watch the stream from twitch.tv everything is fine but when I use twitch GUI with VLC player bottom of the video is cut. Hard to explain, it is basically smaller in lenght. Here I add an image side by side.
![Untitled](https://user-images.githubusercontent.com/78285554/106372155-e17c1780-637d-11eb-851b-edfefdd41aab.png)

**Comment by bastimeyer** [comment, marked as answer]
Check your VLC config. Streamlink Twitch GUI is just a launcher for Streamlink, and Streamlink doesn't touch the video stream data, as it would require re-encoding it on the fly. Your VLC player is probably cropping off parts of the video via some configuration/setting by accident. Unfortunately I can't help you with this, because I don't use VLC.

- https://www.google.com/search?q=vlc+player+overscan
- https://wiki.videolan.org/VLC_command-line_help/
- https://wiki.videolan.org/VLC_HowTo/Crop/
